course_discovery/apps/edx_elasticsearch_dsl_extensions/distinct_counts/tests/test_query.py

```diff
@@ -0,0 +1,430 @@
+import mock
+import pytest
+from elasticsearch_dsl import DateHistogramFacet, Search, TermsFacet
```

Comment:
Please reorder imported libs.

Reply:
Hmm. What exactly do you mean? I see that these orders are fine.